comptoir-ejb/src/main/java/be/valuya/comptoir/model/commercial/ItemVariantSale.java

```diff
@@ -48,6 +42,9 @@
     @NotNull
     @Nonnull
     private Stock stock;
+    @Column(name = "FORCE_CUSTOMER_LOYALTY")
+    @CheckForNull
+    private Boolean forceCustomerLoyalty;
```

Comment:
Nullable field to mark an item sale as used or not for customer loyalty.
If null, default logic applies